data/serializer/src/main/java/tech/pegasys/teku/api/migrated/BlockRewardData.java

```diff
@@ -14,23 +14,22 @@
 package tech.pegasys.teku.api.migrated;
 
 import java.util.Objects;
-import tech.pegasys.teku.infrastructure.unsigned.UInt64;
 
 public class BlockRewardData {
-  private final UInt64 proposerIndex;
-  private final UInt64 total;
-  private final Long attestations;
-  private final UInt64 syncAggregate;
-  private final UInt64 proposerSlashings;
-  private final UInt64 attesterSlashings;
+  private final int proposerIndex;
+  private final long total;
+  private final long attestations;
+  private final long syncAggregate;
+  private final long proposerSlashings;
+  private final long attesterSlashings;
 
   public BlockRewardData(
-      final UInt64 proposerIndex,
-      final UInt64 total,
-      final Long attestations,
-      final UInt64 syncAggregate,
-      final UInt64 proposerSlashings,
-      final UInt64 attesterSlashings) {
+      final int proposerIndex,
+      final long total,
```

Comment:
the more i think about it the more im convinced we just do our addition inside the class and don't initialise with total, it removes the chance of it being accumulated to the wrong value at some random point in code..